design-documents/graph-ql/coverage/negotiableQuotes.graphqls

```diff
@@ -0,0 +1,166 @@
+type Mutation {
+    requestNegotiableQuote(
+        negotiableQuote: NegotiableQuoteRequestInput!
+    ): RequestNegotiableQuoteOutput! @doc(description:"Generates negotiable quote request")
+
+    setShippingMethodsOnNegotiableQuote(
+        negotiableQuoteShippingMethod: NegotiableQuoteShippingMethodInput!
+    ): setShippingMethodsOnNegotiableQuoteOutput @doc(description:"Set negotiable quote shipping method")
+}
+
+type Customer {
+    negotiable_quote (
+        id: ID!): NegotiableQuote @doc(description: "Get negotiable quote of a customer")
+
+    negotiable_quotes(
+        page_size: Int = 20,
+        current_page: Int = 1,
+        filter: NegotiableQuoteFilterInput
+    ): [NegotiableQuote] @doc(description: "Get negotiable quotes of a customer")
+
+    estimateShippingCostForNegotiableQuote(
+        address: NegotiableQuoteShippingAddressInput!
+    ): [ShippingMethodsOutput]! @doc(description:"Estimate shipping costs for a negotiable quote")
+
+    estimateShippingCostByAddressIdForNegotiableQuote(
+        address_id: String!
+    ): [ShippingMethodsOutput]! @doc(description:"Estimate shipping costs for a negotiable quote by Address ID")
+}
+
+type NegotiableQuote {
+    items: [NegotiableQuoteItem] @doc(description: "Negotiable quote item")
+    attachements: [AttachmentContent] @doc(description: "Negotiable quote Attachements")
+    comments: [NegotiableQuoteComment] @doc(description: "Negotiable quote comments")
+    totals: TotalsOutput! @doc(description: "Negotiable quote totals output")
+    payment_info: NegotiableQuotePaymentInfo! @doc(description: "Negotiable quote payment info.")
+    available_payment_methods: [PaymentMethodsOutput] @doc(description: "Available payment methods")
```

Comment:
Is `available_payment_methods` field used in any customer-facing use cases?

Reply:
Removed this and changed to `available_shipping_methods` (As suggested by you in this https://github.com/magento/architecture/pull/381#discussion_r434760897 )